### PR TITLE
Finished the tickets route.

### PR DESCRIPTION
**Pull request by jessampaio**

### CODE REVIEW

File: express-typescript/src/controllers/tickets.ts [hunk 1]
```diff
@@ -1,39 +1,42 @@
...
-export function getPricedTickets(req: Request, res: Response) {
-  const getPricedTicketsQuery = `SELECT * FROM tickets`
+export function getTicketsRemaining(req: Request, res: Response) {
 
-  database.query(getPricedTicketsQuery, (err, data: any) => {
-    if (err != null) {
-      return res.status(500).json(err)
-    }
+  const getTicketsRemainingQuery = `SELECT MAX(ticket_amount) - SUM(ticket_quantity) as tickets_remaining
+  FROM tickets t, events e
```
Comment (herlitzj) [inline express-typescript/src/controllers/tickets.ts:7]: This should be a join

File: express-typescript/src/controllers/tickets.ts [hunk 2]
```diff
@@ -1,39 +1,42 @@
...
+  WHERE t.event_id = e.event_id
+  HAVING tickets_remaining > 0`
+
+  database.query(getTicketsRemainingQuery, [req.body.eventId, req.body.eventId], (err, data: any) => {
```
[the remark this answers is not in the document]
Comment (jessampaio) [reply express-typescript/src/controllers/tickets.ts]: I fixed it, I think.

File: express-typescript/src/controllers/tickets.ts (hunk 1 above)
Comment (jessampaio) [reply express-typescript/src/controllers/tickets.ts]: I fixed it.

**Review by herlitzj — APPROVED** [review]
looks good

**Pull request merged**